.github/workflows/pull_request_tests.yml

```diff
@@ -5,6 +5,10 @@ on:
       - "main"
   pull_request:
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.ref }}
```

Comment:
This looks good. Can we also do this (or instead) with branch name? That way rapid pushes to e.g. `main` wouldn't kick off multiple builds/deploys.

Reply:
Yeah that's the `|| github.ref` part. If it's not a PR it'll use that.

Reply:
Ah! Nice!